aws/retry/middleware.go

```diff
@@ -139,6 +155,7 @@ func (r Attempt) HandleFinalize(ctx context.Context, in smithymiddle.FinalizeInp
 
 		responseMetadata := awsmiddle.GetResponseMetadata(metadata)
 		attemptClockSkew = responseMetadata.AttemptSkew
+		attemptMetadata.Retryable = true
```

Comment:
`Retryable` should be set higher up probably, e.g. result of line 126. This is where the `Retried` member should be set instead, since by fact of reaching this point the AttemptMiddleware will loop again.

This helps distinguish the difference between `Retryable` => `May be retried`, and `Retried` => `Was retried`.